src/sinks/aws_cloudwatch_logs/request.rs

```diff
@@ -169,15 +173,22 @@ impl Future for CloudwatchFuture {
 
                     let next_token = res.next_sequence_token;
 
-                    info!(message = "putting logs was successful.", ?next_token);
+                    if let Some(events) = self.events.take() {
+                        info!(message = "putting logs.", ?next_token);
```

Comment:
Is this necessary? Or should it be moved to `debug`? Don't we already have request logs?

Reply:
> Don't we already have request logs?

We have, but this will be called only in rare scenarios so I would like to make it clear when it happens, and I agree that `info` is too strong for this, `debug` would better fit it's purpose.